src/main/java/com/exadel/discount/dto/favorite/FavoriteDto.java

```diff
@@ -0,0 +1,16 @@
+package com.exadel.discount.dto.favorite;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FavoriteDto {
+    @NotNull
+    private UUID id;
```

Comment:
The same about DiscountDTO as for Coupon here

Reply:
Done